src/commands/mention/dump.ts

```diff
@@ -14,7 +14,10 @@ const command: MentionCommand = {
       content: `💾 Guild data for server ${guildId} as of <t:${Math.floor(now / 1000)}:R>.`,
       files: [{ attachment: Buffer.from(data), name: `Countr-Dump-${guildId}-${now}.json` }],
     })
-      .then(() => reply(`📨 Guild data of ${guildId === message.guildId ? "this server" : `server with ID \`${guildId}\``} has been dumped to your DMs.`))
+      .then(msg => reply(`
+        📨 Guild data of ${guildId === message.guildId ? "this server" : `server with ID \`${guildId}\``} has been dumped to your DMs.
+        [Jump to Message](${msg.url})
```

Comment:
message url should be modified by replacing user's id with bot's id

Reply:
otherwise it will show "you dont have access to that channel/message" pop up

Reply:
there's no user id in the message link, all dms are channels and have a channel id